xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-importer/src/main/java/org/xwiki/officeimporter/internal/splitter/DocumentSplitterUtils.java

```diff
@@ -84,22 +83,20 @@ public static NamingCriterion getNamingCriterion(String namingCriterionId, Strin
      * 
      * @param sectionDoc the newly created wiki document corresponding to a section of the original office document
      * @param officeDocument the office document being splitted into wiki documents
+     * @param fileMap Map of files from the original document
      * @return the relocated artifacts
      */
-    public static Set<File> relocateArtifacts(WikiDocument sectionDoc, XDOMOfficeDocument officeDocument)
+    public static Set<File> relocateArtifacts(WikiDocument sectionDoc, XDOMOfficeDocument officeDocument,
```

Comment:
This method doesn't do any relocation. With your changes it's more of a ``getSectionArtifacts``. Also, if you pass ``fileMap`` then you don't need ``officeDocument`` anymore right?

Reply:
Yes, the method adds files subdocument. There is no need to remove files from the original document as it will be discarded anyway.
So, the only necessary change to fix the issue is to leave the file in the original set.
But I also moved the map generation outside the method since it makes no sense to reconstruct it for every document.